sdk/containerregistry/Azure.Containers.ContainerRegistry/samples/Sample03a_SetImageProperties.md

```diff
@@ -0,0 +1,38 @@
+# Azure.Containers.ContainerRegistry Samples - Set Image Properties (sync)
+
+A common use case for Azure Container Registries is to set an image's properties so it can't be overwritten during a lengthy deployment.
```

Comment:
```suggestion
A common use case for Azure Container Registries is to set an image's property so it can't be overwritten during a lengthy deployment.
```

Reply:
In the sample, we're setting both write and delete, so I'll keep the wording the way it is - thanks, though!

Reply:
Oh, I misinterpreted "latest" to be the property rather than the `CanWrite` and `CanDelete` :)